#!/usr/bin/env python

PACKAGE="gazebo_sfm_plugin"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("pedestrian_velocity", double_t,0, "Actor velocity",.8, 0,2)
gen.add("cyclic", bool_t, 0, "Actor will repeat waypoints",False)

exit(gen.generate(PACKAGE, "PedestrianSFMPlugin", "PedestrianSFMPlugin"))